### PR TITLE
Fix in configuration

### PR DESCRIPTION
**Pull request by yvikhlya**
This PR fixes two issues in MOM5 configuration:

- River spread bug in 1/4 degree setup
- Changes reference year in diag_table to 1900, to make different netcdf tools stop complaining about time scale in MOM diagnostic files.

### CODE REVIEW

File: data/geos5/1440x1080/INPUT/input.nml [hunk 1]
```diff
@@ -342,18 +342,18 @@
...
+	river_diffuse_temp=.false.
 /
 
  &ocean_riverspread_nml
 	use_this_module=.true.
 	debug_this_module=.false
 	riverspread_diffusion=.true.
-	riverspread_diffusion_passes=500
-	vel_micom_smooth=2.0
+	riverspread_diffusion_passes=2
```
Comment (sanAkel) [inline data/geos5/1440x1080/INPUT/input.nml:355]: is this being _ported_ from S2S v3?

**Review by sanAkel — COMMENTED** [review]
Are these being _ported_ from S2S v3?

File: data/geos5/1440x1080/INPUT/input.nml (hunk 1 above)
Comment (yvikhlya) [reply data/geos5/1440x1080/INPUT/input.nml:355]: Yes.

**Review by sanAkel — APPROVED** [review]

**Comment by mathomp4** [comment]
@sanAkel @yvikhlya I'm thinking of making a new release of MOM5 because of a simple dumb PR I made (#18).

Should we pull this into the release as well?

**Comment by yvikhlya** [comment]
This was supposed to be merged long time ago.

**Comment by yvikhlya** [comment]
> @sanAkel @yvikhlya I'm thinking of making a new release of MOM5 because of a simple dumb PR I made (#18).
> 
> Should we pull this into the release as well?

Yes, sure.

**Comment by mathomp4** [comment]
@yvikhlya Query: Is this non-zero-diff for MOM5?

**Comment by yvikhlya** [comment]
> @yvikhlya Query: Is this non-zero-diff for MOM5?

This is zero diff in all resolutions, except 1/4 degree. So, technically, no this is not exactly zero diff.

**Comment by mathomp4** [comment]
> > @yvikhlya Query: Is this non-zero-diff for MOM5?
> 
> This is zero diff in all resolutions, except 1/4 degree. So, technically, no this is not exactly zero diff.

Thank you!

**Pull request merged**